hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/realtime/HoodieRealtimeFileSplit.java

```diff
@@ -58,6 +62,10 @@ public HoodieRealtimeFileSplit(FileSplit baseSplit, String basePath, List<String
     return deltaLogPaths;
   }
 
+  public List<FileStatus> getDeltaLogFileStatus() {
```

Comment:
Since `FileStatus` has path as well, do you think we still need `getDeltaLogPaths` API? Removing that API touches a few other parts as well, so I am not suggesting to take it up in this patch. But, if you think that's ok then we can track it in a JIRA.

Reply:
Already we have a tracking ticket on this end: https://issues.apache.org/jira/browse/HUDI-2613